src/Options/index.js

```diff
@@ -153,7 +153,7 @@ export interface ParseServerOptions {
   /* Max file size for uploads, defaults to 20mb
   :DEFAULT: 20mb */
   maxUploadSize: ?string;
-  /* Set to `true` to require users to verify their email address to complete the sign-up process.
+  /* Set to `true` to require users to verify their email address to complete the sign-up process. Supports a function with a return value of `true` or `false` for condiitonal verification.
   <br><br>
   Default is `false`.
   :DEFAULT: false */
```

Comment:
If this supports also a function, should the type reflect that?

Reply:
See https://github.com/parse-community/parse-server/pull/8425#discussion_r1200603530